keel-web/src/test/kotlin/com/netflix/spinnaker/keel/apidocs/ApiDocTests.kt

```diff
@@ -95,26 +93,20 @@ class ApiDocTests : JUnit5Minutests {
     VersionedTagProvider::class
   )
 
-  @Autowired
-  lateinit var artifactSuppliers: List<ArtifactSupplier<*, *>>
-
   val artifactTypes
-    get() = artifactSuppliers.map { it.supportedArtifact.artifactClass.kotlin }
-
-  val api: JsonNode by lazy {
-    mvc
-      .perform(get("/v3/api-docs").accept(APPLICATION_JSON_VALUE))
-      .andExpect(status().isOk)
-      .andReturn()
-      .response
-      .contentAsString
-      .also(::println)
-      .let { jacksonObjectMapper().readTree(it) }
-  }
+    get() = extensionRegistry.extensionsOf<DeliveryArtifact>().values.toList()
 
   // FIXME: the order of the @Beans used to customize type handling for the OpenAPI stuff is causing a stack overflow
```

Comment:
If you got this to pass, please remove the `FIXME` comment.

Reply:
I bet if I do the tests will fail again :sob: